packages/devtools_app/lib/src/server_api_client.dart

```diff
@@ -78,7 +79,18 @@ class DevToolsServerApiClient {
       return;
     }
 
-    Notification('Dart DevTools',
+    // Dismiss any earlier notifications first so they don't build up
+    // in the notifications list if the user presses the button multiple times.
+    dismissNotifications();
+
+    _lastNotification = Notification('Dart DevTools',
         body: 'DevTools is available in this existing browser window');
   }
+
+  void dismissNotifications() {
+    if (_lastNotification != null) {
+      print('Closing notification!');
```

Comment:
Please use the `log` method from `logger.dart` for this so you can indicate a log level.

Reply:
Er whoops, this was a temporary debug print to track down why the code wasn't called. I'll remove it - thanks for spotting! 😄